tools/add-fpr-rule

```diff
@@ -1,4 +1,22 @@
-#!/usr/bin/python
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+# This file is part of Archivematica.
```

Comment:
Is it? Or is dev-tools considered a separate project for legal headers like this?